### PR TITLE
Dropdown menu

### PR DESCRIPTION
**Pull request by mruwnik**
+ https://github.com/StampyAI/stampy-ui/issues/331 while I was at it

image

image

image

image

image

image

### CODE REVIEW

File: app/root.css [hunk 1]
```diff
@@ -267,6 +271,46 @@ svg * {
...
+  background-color: var(--bgColorPage);
+}
+
+.menu-link:first-child {
+  border-top: none;
+}
+
+.menu-link:last-child {
+  border-bottom: none;
+}
```
Comment (mruwnik) [inline app/root.css:312]: I really hope someone will come along and do proper styling of everything, so I don't have to...

File: app/components/icons-generated/Followup.js [hunk 2]
```diff
@@ -16,10 +16,8 @@ const SvgFollowup = (props) => (
       }}
     />
     <path
+      className="followup_svg__st1 followup_svg__gray"
       d="M283.7 127.6h-60.6c-30.4 0-55.1 24.7-55.1 55 0 4.4 3.6 8 8 8s8-3.6 8-8c0-21.5 17.5-39.1 39.1-39.1h60.6c24.4 0 44.3 19.9 44.3 44.3 0 16.9-9.4 32.1-24.6 39.7L252.3 254c-2.6 1.4-4.3 4.2-4.3 7.1V288c0 4.4 3.6 8 8 8s8-3.6 8-8v-21.9l46.7-24.2c20.5-10.3 33.3-31 33.3-54 0-33.3-27.1-60.3-60.3-60.3zM256 328c-8.8 0-16 7.2-16 16s7.2 16 16 16 16-7.2 16-16-7.2-16-16-16zm0-296.9C114.6 31.1.9 124.2.9 239.1c0 49.6 21.4 95 57 130.7-12.5 50.4-54.3 95.3-54.8 95.8-2.2 2.3-2.8 5.7-1.5 8.7s4.1 4.8 7.3 4.8c66.3 0 116-31.8 140.6-51.4 32.7 12.3 69 19.4 107.4 19.4 141.4 0 255.1-93.1 255.1-208s-114.6-208-256-208zm0 400c-35.1 0-69.4-6.2-101.8-18.4l-8.5-2.3-7.1 5.7c-20.2 16.1-60.1 42-112.7 47.1 14.4-18.3 37.4-51.7 46.6-88.6l2.2-8.8-6.4-6.4C34.1 325.1 16 283.8 16 239.1c0-105.9 107.7-192 240-192s240 86.1 240 192-107.7 192-240 192z"
-      style={{
-        fill: '#333',
```
Comment (mruwnik) [inline app/components/icons-generated/Followup.js:21]: this now uses the default fill colour, which will handle light/dark mode gracefully

**Comment by mruwnik** [comment]
should I just deploy this to a test server, rather than uploading millions of pictures?

File: app/root.css (hunk 1 above)
Comment (Aprillion) [reply app/root.css:312]: IMHO if there is no link to GH issue description and/or Figma, then there is no "proper" styling to implement 🤷

**Comment by Aprillion** [comment]
> should I just deploy this to a test server

yes

File: app/assets/icons/followup.svg
Comment (Aprillion) [inline app/assets/icons/followup.svg:1]: I'm not sure about the reversal of "asking for help" icon, the usual meaning is that the user asks for help and the site is helpful ... here the site is asking for help and the user is supposed to be helpful 🤔 

How is the user supposed to know that meaning before clicking on the icon? Was this discussed on some meeting? What was the answer on that meeting, please?

File: app/root.css (hunk 1 above)
Comment (ccstan99) [reply app/root.css:312]: From #285 hidden to the far right of linked [Figma file](https://www.figma.com/file/ipifwmO1KYfqakpl3srFIL/2023-08-08-AI-Safety-ccc-ccstan99?type=design&node-id=90-2047&mode=design)
Screenshot 2023-11-02 at 1 19 18 AM
Screenshot 2023-11-02 at 1 21 27 AM

**Review by Aprillion — DISMISSED** [review]

**Review by Aprillion — APPROVED** [review]

**Pull request merged**